src/Evolution/Systems/GrMhd/ValenciaDivClean/KastaunEtAlHydro.cpp

```diff
@@ -71,6 +74,7 @@ class FunctionOfZ {
   bool has_no_root() { return state_is_unphysical_; };
 
  private:
+  const double lorentz_max_ = 1000.;
```

Comment:
why are `lorentz_max_`, `v_0_squared_`, and `kmax_` member variables?

Reply:
I don't know, I didn't right the original code.

Reply:
Well, I am asking it to be changed as it initially looked to me that `lorentz_max_` is hard-coded to be a 1000